src/plugins/intel_gpu/src/plugin/transformations_pipeline.cpp

```diff
@@ -716,7 +718,14 @@ void TransformationsPipeline::apply(std::shared_ptr<ov::Model> func) {
             manager.register_pass<ov::pass::ConstantFolding>();
         }
         manager.register_pass<ov::pass::ConvertGatherToGatherCompressed>();
-        manager.register_pass<ov::intel_gpu::RMSFusion>(device_info.max_work_group_size);
+        auto pass_config = manager.get_pass_config();
+        pass_config->set_callback<ov::pass::RMSFusion>([=](const_node_ptr& root) -> bool {
+            const auto& gamma_shape = root->get_input_node_ptr(0)->get_input_partial_shape(0).to_shape();
```

Comment:
should we check that 
`root->get_input_node_ptr(0)->get_input_partial_shape(0)`
returns static shape.
`.to_shape();` throws exception in case of dynamic shape

Reply:
maybe it's better to take the last partial shape dim and check that it's static and get its static value to use

Reply:
done